packages/shared/styles/components/SfCollectedProduct.scss

```diff
@@ -0,0 +1,68 @@
+@import "../variables";
+
+$collected-product-hover-shadow: 0px 4px 35px rgba(168, 172, 176, 0.19) !default;
+$collected-product__remove-bg: $c-gray-secondary !default;
+.sf-collected-product {
+  max-width: 100%;
```

Comment:
I believe it shouldn't be 100%, rather we should have full-width modifier, wdyt?

Reply:
i think it's great, i talked to @leomp12  earlier it and he had asked to use it as full-width because he didn't know if a value had been proposed